openfermionpyscf/_molecular_data.py

```diff
@@ -0,0 +1,123 @@
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+
+"""Class to use pyscf program to access quantum chemistry data."""
+
+from functools import reduce
+
+import numpy
+from openfermion.hamiltonians import MolecularData
+from pyscf import ao2mo
+
+
+class MolecularData(MolecularData):
```

Comment:
Please change the name of this class. PyscfMolecularData would be fine, for instance.